# GEO-submissions

Here are some notes about GEO submissions; some are more general, others are specifically tailored towards submissions at the Babraham Institute.

https://www.ncbi.nlm.nih.gov/geo/info/seq.html


To submit to these repositories you'll need all of the raw (fastq) sequence data, plus some kind of quantitation.  The quantitation can be anything, but we'd normally use per gene expression measures for RNA-Seq, peak quantitation for ChIP and percent methylation over whatever windows you used for methylation data.

## The metadata spreadsheet

## Submission from Babraham

- contact Babraham Bioinformatics with a list of files, the sample name as well as the desired name of the sample for the manuscript, e.g.:

```
SierraID      sample name on Sierra                name in manuscript
4777           lane4777_TAGCATAG_GAGAGTCT_....        Dnmt1_WT_rep1
...
```

- we will take a look, collate all necessary files and return a 
- please do 
https://www.ncbi.nlm.nih.gov/geo/info/examples/seq_template.xlsx

## Quantified files

### Bisulfite-seq 

This is fairly straight forward, both Bismark coverage files (ending in `.cov.gz`) or genome-wide cytosine reports are acceptable. Coverage files are preferred as they are generated by default, less unwieldy and more precise (see here for differences between the two different file formats: [difference between coverage and cytosine report files](https://github.com/FelixKrueger/Bismark/blob/master/Docs/FAQ.md#context-changediscrepancy-between-bismark-coverage-and-genome-wide-cytosine-reports)

### RNA-seq

### ChIP-seq/ATAC-seq etc

