# Notes regarding GEO submissions

Here are some notes about GEO submissions; some are more general, others are specifically tailored towards submissions at the Babraham Institute.

The entire submission process is described in detail over at the [GEO high-throughput sequencing data submission](https://www.ncbi.nlm.nih.gov/geo/info/seq.html) pages, here we focus on the more practical implications. While GEO handles the metadata and quantitation of your submission, the raw sequencing data itself is handled by the [Sequencing Read Archive](https://www.ncbi.nlm.nih.gov/sra), and will be available via the [NCBI](https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?), the European [ENA](https://www.ebi.ac.uk/ena/browser/home) or the Japanese [DDBJ](https://www.ddbj.nig.ac.jp/dra/index-e.html). For a quick look and easy way to retrieve raw data from the SRA take a look at the [SRA-Explorer](https://sra-explorer.info/).

To submit sequencing data to the GEO repository you will need:
 - raw sequencing data (`FastQ` files), as well as 
 - some kind of application-sepcific quantitation (a `BAM` file is not sufficient)
 - a metadata spreadsheet detailing the experiment and how the files are related to each other
 
 
 The quantitation can be anything, but we'd normally use per gene expression measures for RNA-Seq, peak quantitation for ChIP and percent methylation over whatever windows you used for methylation data.

## The metadata spreadsheet

## Submission from Babraham

- contact Babraham Bioinformatics with a list of files, the sample name as well as the desired name of the sample for the manuscript, e.g.:

```
SierraID      sample name on Sierra                name in manuscript
  4777     lane4777_TAGCATAG_GAGAGTCT_....           Dnmt1_WT_rep1
...
```

- we will take a look, collate all necessary files and return to you a the metadata spreadsheet (see above) to fill in the details.
- please **do not** start by filling in the metadata spreadsheet with filenames and details, as this will make it much harder for us to produce the spreadsheet, and is also considerably more prone to errors.

https://www.ncbi.nlm.nih.gov/geo/info/examples/seq_template.xlsx

## Quantified files

### Bisulfite-seq 

This is fairly straight forward, both Bismark coverage files (ending in `.cov.gz`) or genome-wide cytosine reports are acceptable. Coverage files are preferred as they are generated by default, less unwieldy and more precise (see here for differences between the two different file formats: [difference between coverage and cytosine report files](https://github.com/FelixKrueger/Bismark/blob/master/Docs/FAQ.md#context-changediscrepancy-between-bismark-coverage-and-genome-wide-cytosine-reports)

### RNA-seq

### ChIP-seq/ATAC-seq etc

### Reviewer token



